lib/mojito.js

```diff
@@ -1,15 +1,18 @@
 Mojito = (function () {
-    // Enable debugging through the console
-    var Options = {
+    var Defaults = {
```

Comment:
I think `Defaults` or `defaultOptions` is a more descriptive choice of variable name here.